boards/arm/mimxrt1170_evk/mimxrt1170_evk_cm7.dts

```diff
@@ -75,7 +75,7 @@ zephyr_mipi_dsi: &mipi_dsi {
 &flexcan3 {
 	status = "okay";
 	bus-speed = <125000>;
-
+	bus-speed-data = <1000000>;
```

Comment:
Was `can-fd` tested?

Reply:
We did test it, It's been a while though. I can flash and test again.

Reply:
It comes from needing to set FD compatibility in the nxp_rt11xx.dtsi

Reply:
And once that gets added you have to include that in the higher level mimxrt1170_evk_cm7.dts as well.

Reply:
> We did test it, It's been a while though. I can flash and test again. That value was selected as the default as it can be used as well with a smaller DLC to send CAN2.0 frames over FD.

If you made a lot of changes to the branch since testing it then please test again